### ISSUE TITLE
a11y: Color Contrast - Primary Navigation

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13460]
Insufficient color contrast for standard text

## Instances

Foreground color: #3D98D3
Background color: #FFFFFF
Contrast ratio:  3.16:1
Primary Navigation 
- [ ] local
- [ ] vpat-testing-downstream
- [ ] Cluster Management
- [ ] Continuous Delivery
- [ ] Virtualization Management and so on

Foreground color: #FFFFFF
Background color: #297DB4
Contrast ratio: 4.48:1
Primary Navigation
- [ ] Home (In selected state) 
- [ ] local, vpat-testing-downstream, Cluster Management, Continuous Delivery (On hover state) and so on

## Actual result

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Expected result

The contrast ratio between foreground text and background for the standard text should be sufficient.

## Recommendation

On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text.

### Light

![Image](https://github.com/user-attachments/assets/77dce1f5-dec2-4321-a171-566f0e9b562a)
![Image](https://github.com/user-attachments/assets/1e205785-ec77-4f0d-9df4-618b41c5c82f)
![Image](https://github.com/user-attachments/assets/63b2092f-6b5b-416e-acfe-58ccbf015cb3)
![Image](https://github.com/user-attachments/assets/4fc090be-a961-4dd4-a6fa-9bf1f8358ef5)
![Image](https://github.com/user-attachments/assets/9a28d741-baf7-470d-b07e-8d49adcc120a)

### Dark

![Image](https://github.com/user-attachments/assets/a96950ce-28dc-4699-ba4f-b038dd2f4d9d)

**Comment by rak-phillip** [comment]
Reported as fixed in VPAT. Issue 15
